.pylintrc-tests

```diff
@@ -1,5 +1,10 @@
 [MESSAGES CONTROL]
-disable=missing-docstring,import-outside-toplevel,duplicate-code,unused-argument,pointless-statement
+disable=missing-docstring,
+        import-outside-toplevel,
+        duplicate-code,
+        unused-argument,
+        pointless-statement,
+        consider-using-f-string # Keep .format() as long as we want legacy python3.5 support
```

Comment:
Minor: I think you should drop the `legacy` word, I feel it adds nothing except a bit of a negative tone.

Reply:
I agree, it's a bit negative.
I felt like I couldn't just say `as long as we want python3.5 support` since we don't officially support python3.5 anymore. I wanted some modifier to reflect that. What about `unofficial python3.5 support`?

Reply:
How about `# Keep .format() as long as we support python3.5`?